content/docs/latest/spire/understand/concepts.md

```diff
@@ -88,12 +88,26 @@ This bootstrap bundle is a default configuration, and should be replaced with cu
 10. The server performs node resolution, to verify additional properties about the agent node and update its registration entries accordingly. For example, if the node was attested using  Microsoft Azure Managed Service Identity (MSI). The resolver extracts the Tenant ID and Principal ID from the agent SPIFFE ID and uses the various Azure services to get information for building an additional set of selectors.
 11. The server issues an SVID to the agent, representing the identity of the agent itself.  
 12. The agent contacts the server (using its SVID as its TLS client certificate) to obtain the registration entries it is authorized for and to ask the server to sign workload SVIDs.
-13. Now fully bootstrapped, the agent turns on the Workload API.  
-14. A workload calls the Workload API to request an SVID.
-15. The agent initiates the workload attestation process by calling its workload attestors, providing them with the process ID of the workload process.  
-16. Attestors use the process ID to discover additional information about the workload
-17. The attestors return the discovered information to agent in the form of selectors
-18. The agent determines the workload's identity by comparing discovered selectors to registration entries, and returns the correct SVID to the workload when the workload asks for it.  
+13. The server sees the SPIFFE ID of the agent's own SVID (the one used as it's TLS client certificate), and fetches all [authorized registration entries](#authorized-registration-entries). All Those are the workload entries signed and return to the agent.
```

Comment:
Well, true! Also fixed the original version: bd737ed04393f3611b8a05a7efc943f5eb81a616

Reply:
I think the the agent's authentication of the server via bootstrap certificates is (also or already?) done between step 11. & 12.?

Reply:
@dfeldman 9ea6bdf7b6f0f4a8151964a806268957eb672611 I think it is clearer, now. Thanks! What's missing is when the SVIDs are generated. Since its _an average day in the life of a SVID_ &mdash; but a SVID isn't formally born and the original version was not correct.

Reply:
Ah sorry, your diagram clarified.

Reply:
I hope this is accurate, now: f07d49b375305fb939ea562d7593b715c4614ead